pilot/pkg/features/pilot.go

```diff
@@ -314,6 +314,10 @@ var (
 	EnableServiceEntrySelectPods = env.RegisterBoolVar("PILOT_ENABLE_SERVICEENTRY_SELECT_PODS", true,
 		"If enabled, service entries with selectors will select pods from the cluster. "+
 			"It is safe to disable it if you are quite sure you don't need this feature").Get()
+
 	InjectionWebhookConfigName = env.RegisterStringVar("INJECTION_WEBHOOK_CONFIG_NAME", "istio-sidecar-injector",
 		"Name of the mutatingwebhookconfiguration to patch, if istioctl is not used.")
+
+	EnableProxyDynamicStats = env.RegisterBoolVar("PROXY_DYNAMIC_STATS", true,
```

Comment:
`PILOT_ENABLE_PROXY_DYNAMIC_STATS`?

Reply:
They are actually cluster stats so I changed to `PILOT_ENABLE_CLUSTER_DYNAMIC_STATS`